### PR TITLE
[#27] Add usage documentation

### PR DESCRIPTION
**Pull request by Marco-Prontera**
The purpose is to add documentation for using the library for creating a custom CMP.

Issue code: [#27]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,164 @@
...
+    TCStringService,
+    ACStringService
+} from "@pubtech/solo-cmp";
+
+import DependencyInjectionManager from "@pubtech/solo-cmp/lib/DependencyInjection/DependencyInjectionManager";
+
+/**
+As you can see in the render functions, a 'rootElement' parameter is provided, this is the HTMLElement 
+that corresponds to the id provided, this allows the library to check whether or not the HTMLElement with 
+that id exists, if not existed creates it for you and places it inside the body of the site.
```
Comment (eugcar) [inline README.md:39]: _exists, if not exists then it creates it for you placing it inside_

**Review by lucop1 — APPROVED** [review]
Seems clear to me

File: README.md (hunk 1 above)
Comment (Marco-Prontera) [reply README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -1,6 +1,164 @@
...
+    cmpVersion,
+    cmpVendorListVersion,
+    'euconsent-v2', //Cookie name used to save TCString in browser
+    'ac_euconsent-v2', //Key name used to save ACString in browser (local storage)
+    cmpId,
+    true, //isServiceSpecific flag this should be used to configure exactly the library of IAB.
+    "https://cdn.pubtech.ai/vendorList", //The url where vendorlist.json if fetched and also for google-vendor-list.json (used to create an ACString).
+    '30vh', //Amp configuration this configure the initial height of the CMP running in an AMP environment.
+    false //Amp configuration this configure the border of the CMP running in an AMP environment.
+);
```
Comment (eugcar) [inline README.md:89]: Wouldn't be better if this one is an object with key-value instead of a list of arguments?

**Review by eugcar — APPROVED** [review]
Ok to me 💪🏼

**Pull request merged**